packages/app/src/vis-packs/ValueLoader.module.css

```diff
@@ -34,6 +46,25 @@
   background-color: var(--primary);
 }
 
+.progress:indeterminate::-webkit-progress-value {
+  width: 100% !important;
+  background-image: var(--indeterminate-bg);
+  background-size: 3rem 3rem;
+  animation: stripes 1s linear infinite;
```

Comment:
Full disclosure, I couldn't get this `animation` to work in Chrome (the stripes are static). I've left the code as is hoping that it's just a passing bug in Chrome... Hopefully we'll have a cross-browser way to style progress bars at some point...

Reply:
Could you remove the progress bar altogether in this case ?

Reply:
It's just the animation; the stripes still give a sense that something is ongoing (and better than no progress bar or an empty progress bar like before, in my opinion):

[Screencast from 2025-05-20 15-36-32.webm](https://github.com/user-attachments/assets/02e1c36a-caa2-4746-bc04-61fcac165772)